### DISCUSSION TITLE
ND-WinningParty xml creation

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#433] [Q&A] [answered]
Good Morning lab !
for this node , watching its path predicate, it seems that it will exist, only if its xpath predicate is true.
Can you tell me please, if that is true, and if yes, if it is a general rule for all nodes  ?
Thanks a lot
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
Predicates are normally used to add some context and allow to differentiate nodes which may follow different rules (e.g. repeatability) and this wouldn't be possible in their absence. 
For the ND-WinningParty specific case, the node is parent of few fields while it should be the same parent node as other organization related fields. The rule should be moved to the set of conditionally applied rules relative to the fields existence instead.
